Fix a race condition with ninja >= 1.12.0 where included headers have not
yet been generated by uic.

--- src/tabs/CMakeLists.txt.orig	2020-09-06 07:44:34 UTC
+++ src/tabs/CMakeLists.txt
@@ -22,3 +22,4 @@ target_include_directories(tabs PUBLIC ${CMAKE_CURRENT
 add_library(tabs STATIC  ${HEADERS} ${MOC_SOURCES} ${PLAIN_SOURCES})
 target_link_libraries(tabs ${QT_LIBRARIES} ${iris_LIB} tools widgets)
 target_include_directories(tabs PUBLIC ${CMAKE_CURRENT_SOURCE_DIR} ${CMAKE_CURRENT_BINARY_DIR})
+add_dependencies(tabs build_ui_files)
